### ISSUE TITLE
Add warning if config is marked as untested

### ISSUE THREAD

**Issue by MagicLike** [openandroidinstaller-dev/openandroidinstaller#114]
For experimental or untested configs it would be nice to have a config metadata to mark it as experimental/untested and output a warning in OAI.

**Comment by tsterbak** [comment]
What would be the reason to put a experimental or untested config to the installer?

**Comment by MagicLike** [comment]
If it works not the same every time or if someone makes a config without having the device itself, so others can test and verify it... ([Just like I recently offered someone on fedi to write a config for their device)](https://mstdn.social/@MagicLike/110028196364500989)

**Comment by MagicLike** [comment]
Also the configs are already declared as `tested` or `under development` in the README...

**Comment by tsterbak** [comment]
> Also the configs are already declared as `tested` or `under development` in the README...

Yeah, I will soon try to align information in the README and in the configs (and on the website). Then the configs will be the only source of truth on what and how stuff is supported. Maybe even more details, it also came up here: #116

**Comment by anon1892** [comment]
Could also be done with #221

**Comment by MagicLike** [comment]
> Could also be done with #221 

Not quite, but mayve 🤔

**Comment by anon1892** [comment]
It can look like that :

![image](https://github.com/openandroidinstaller-dev/openandroidinstaller/assets/40572253/034a37b7-5571-4bb0-ad0d-96b68db8b73e)

**Comment by MagicLike** [comment]
> It can look like that :
> 
> ![image](https://github.com/openandroidinstaller-dev/openandroidinstaller/assets/40572253/034a37b7-5571-4bb0-ad0d-96b68db8b73e)
> 

Could you also add the functionality to just flag configs as untested and then this warning in red pops up and the user needs to co firm twice to continue?

**Comment by anon1892** [comment]
I don't think it's really useful, since you just have to put this in `metadata`
```
notes: >
  - **This device has never been tested with OAI.** etc.
```
Instead of
```
untested: true
```

I think `notes` is more versatile than add another field. It's already in red and bold, users have to read… What do you think?

**Comment by MagicLike** [comment]
> I don't think it's really useful, since you just have to put this in `metadata`
> ```
> notes: >
>   - **This device has never been tested with OAI.** etc.
> ```
> Instead of
> ```
> untested: true
> ```
> 
> I think `notes` is more versatile than add another field. It's already in red and bold, users have to read… What do you think?

I think that it is better to convert as much as possible into fileds / variables, to ensure continuety and avoid spelling issues, as I occasionally find spelling issues in every config now and then... I mean another field is not neccesary, but would just look nice. Also all the warnings and notes you wrote for the Xiaomi devices could be hard coded and just set by `is_xiaomi: true. (Or something like that.) It also makes the configs more simple and not that crammed.

**Comment by anon1892** [comment]
Ok, I see. Will work on it.

**Comment by MagicLike** [comment]
> Maybe should we consider #72 ?

I think this is some work for later, as @tsterbak stated - he wants OAI to have all the features and major stuff imolemented first, before getting into translating it.

**Comment by MagicLike** [comment]
resolved with #221
